security-questionnaire.md

```diff
@@ -0,0 +1,50 @@
+| [3.4 Does this specification expose persistent, cross-origin state to the web?]                             | No                                                               | No            | No            | No            | No            | No            | No            |
+| [3.5 Does this specification expose any other data to an origin that it doesn’t currently have access to?]  | No                                                               | No            | No            | No            | No            | No            | No            |
+| [3.6 Does this specification enable new script execution/loading mechanisms?]                               | No                                                               | No            | No            | No            | No            | No            | No            |
+| [3.7 Does this specification allow an origin access to a user’s location?]                                  | Depends on concrete sensor class.                                | Indirectly    | No            | Indirectly    | Indirectly    | Day / Night?  | Unknown       |
+| [3.8 Does this specification allow an origin access to sensors on a user’s device?]                         | Yes                                                              | Yes           | Yes           | Yes           | Yes           | Yes           | Yes           |
+| [3.9 Does this specification allow an origin access to aspects of a user’s local computing environment?]    | Yes. The API provides means to check what sensors are available. | Yes           | Yes           | Yes           | Yes           | Yes           | Yes           |
+| [3.10 Does this specification allow an origin access to other devices?]                                     | No                                                               | No            | No            | No            | No            | No            | No            |
+| [3.11 Does this specification allow an origin some measure of control over a user agent’s native UI?]       | No                                                               | No            | No            | No            | No            | No            | No            |
+| [3.12 Does this specification expose temporary identifiers to the web?]                                     | No                                                               | No            | No            | No            | No            | No            | No            |
+| [3.13 Does this specification distinguish between behavior in first-party and third-party contexts?]        | No. Sensors are exposed only to top level browsing context.      | No            | No            | No            | No            | No            | No            |
```

Comment:
Doesn't it mean `yes` actually?

Reply:
I was thinking about it, 3.13 talks about Section 2.1 of [FIRST-PARTY-ONLY](https://tools.ietf.org/html/draft-west-first-party-cookies-07). I would say no, since we don't check first / third party 'contexts' as specified in [FIRST-PARTY-ONLY](https://tools.ietf.org/html/draft-west-first-party-cookies-07).

API is not checking whether SameSite is set, e.g.,:
`Set-Cookie: SID=31d4d96e407aad42; SameSite=Strict`

Reply:
I can remove misleading text:
>Sensors are exposed only to top level browsing context.

wdyt?

Reply:
yep, let's do so